README.md

```diff
@@ -1 +1,44 @@
-# bitrise-step-activate-build-cache-for-bazel
\ No newline at end of file
+# Activate Bitrise Build Cache for Bazel
+
+[![Step changelog](https://shields.io/github/v/release/bitrise-steplib/bitrise-step-activate-build-cache-for-bazel?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-activate-build-cache-for-bazel/releases)
+
+Activates Bitrise Remote Build Cache for subsequent Bazel builds in the workflows.
+
+
+<details>
+<summary>Description</summary>
+
+This Step activates Bitrise's remote build cache add-on for subsequent Bazel executions in the workflow.
+
+After this Step executes, Bazel builds will automatically read from the remote cache and push new entries if it's enabled.
+
+</details>
+
+## 🧩 Get started
```

Comment:
Do we want to drop a line somewhere about the CLI? @viktorbenei ?

Reply:
can do in a next changeset